### PR TITLE
feature/no_account

### PR DESCRIPTION
**Pull request by pgilliar**
Added the possibility to access the app without creating an account but with limited accessibility

### CODE REVIEW

File: app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt [hunk 1]
```diff
@@ -42,6 +43,7 @@ import com.google.firebase.ktx.Firebase
 import kotlinx.coroutines.launch
 
 const val CREATE_ACCOUNT_PROFILE = "com.github.palFinderTeam.palFinder.CREATE_ACCOUNT_PROFILE"
+var IS_NO_ACCOUNT_USER = false
```
Comment (zaclegarssure) [inline app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt:46]: Not necessary a good practice, maybe you could put it as a member of ProfileService and have way to set the service in no account mode or not.
On the other hand this is a simple solution.

Comment (zaclegarssure) [reply app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt:46]: you could even try if simply checking that currentLoggedInUser == null <=> IS_NO_ACCOUNT_USER

**Comment by zaclegarssure** [comment]
Also if you connect after the pop up show you will still be considered disconnected.

**Review by TruddyTheDuddi — APPROVED** [review]
Looks good! The addition of a popup function is good, maybe change the file name from `NoAccountWarning.kt` to `PopupCreator` for the name to be generalized.

**Review by zaclegarssure — APPROVED** [review]
Okay nice.

**Pull request merged**